### PR TITLE
Refactor selections

### LINKED ISSUE #465: UX: Profile name should be shown in the connection screen
# Installation
How did you install the eduVPN client? 

-> Via the repository

# Version
What version of the client you are running? Try your package manager or otherwise `$ eduvpn-client -v`.

-> 2.2.1-1

# OS/Distribution
What operating system/distribution and version you are running?

-> Ubuntu 20.04.4

## Do you have a problem during connecting?
Sometimes (ege. when using the back button with an already active VPN) clicking in the free space below the profile list an unidentifiable profile is selected and connected. 

Also the Name is not shown on the connection screen.

### PR DESCRIPTION
**Pull request by jwijenbergh**
Fixes #465 

This has a bunch of benefits to the way we did it before. In short it replaces the *selection* signals into *activated* signals which is the proper way to do things. For this, we need to set up the needed signals, change the activated signal from double click to signal click. Additionally, it also sets selection hover so that the user can easily see which entry would be selected on a click!

So in short, we get the following improvements:
- We can now see the selection before it gets activated
- No more buggy selections because the user actually has to click on it instead of clicking *anywhere*
- Way simpler code

Draft because I want at least a confirmation from 1 other maintainer that this works well, and I need to test it on some vms.

### CODE REVIEW

**Comment by jwijenbergh** [comment]
Works fine
